test/util/create_utils.go

```diff
@@ -114,3 +114,29 @@ func CreateTestTemplateVersions(numTemplateVersions int, ctx context.Context, tv
 	}
 	return nil
 }
+
+func CreateRepositories(numRepositories int, ctx context.Context, storeInst store.Store, orgId uuid.UUID) error {
```

Comment:
minor: ctx first param :pray:

Reply:
Yep.  Fixed this one and a few other helper methods.